portfolio.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>My Portfolio</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>	
+	
+	<body>
+		
+		<nav class="nav_bar">
+				<p>Karen Schulz</p>
```

Comment:
i would probably use a h tag here for your name

Reply:
hummm I will see... will play with it but if it doesnt look ok will leave.

Reply:
I think unless you see the actual page, it is difficult to assess